spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java

```diff
@@ -56,6 +56,12 @@
 	static final String INDEXED_PROPERTY_KEY = PREFIX + "indexed";
 
 	/**
+	 * Constant property key to be used in the deployer implementation as an environment variable
```

Comment:
Note that I'm not saying "deployment property", because this is to be set at each app level *by* the deployer, as opposed to deployment properties that are *read* by the deployer